Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs

```diff
@@ -509,9 +608,16 @@ public void UseHttp(IApplicationBuilder app)
             else
             {
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+                this.Logger.LogInformation("Using HSTS, which sets Strict-Transport-Security Header");
                 app.UseHsts();
             }
 
+            if (!this.environment.IsDevelopment())
+            {
+                app.UseHttpsRedirection();
+            }
+
+            app.UseStaticFiles();
```

Comment:
I dont think use static files is necessary for this method since its used on the applications that dont serve static content

Reply:
I didn't add that, it was there before. AdminWebClient may use this common code and it does have static files. It doesn't seem to have any negative affect on Hangfire.

Reply:
I had removed it, then added it back. The main thing I added was to not use Https redirection when in Development mode.

Reply:
thats what I saw. UseStaticFiles is used in Admin/Startup. I think we can remove it here

Reply:
ok sounds good. I just worry about ordering sensitivity as this seems to be fussy on ordering, at least services configuration sure is!

Reply:
In particular everywhere I look the app.Use* ordering is the same, with UseStaticFiles coming before UseRouting. I worry that this order if messed up will create side effects.

Reply:
i.e. https://docs.microsoft.com/en-us/aspnet/core/fundamentals/middleware/?view=aspnetcore-3.1#middleware-order